Dockerfile

```diff
@@ -1,20 +1,17 @@
 
-FROM circleci/openjdk:8
+FROM circleci/openjdk:11
 
 RUN set -eux; \
-    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 0x219BD9C9; \
-    echo 'deb http://repos.azulsystems.com/debian stable main' | sudo tee -a /etc/apt/sources.list.d/zulu.list; \
     sudo apt-get update; \
     sudo apt-get dist-upgrade; \
-    sudo apt-get install socat zulu-7 zulu-9 zulu-10 zulu-11 zulu-12;
-
-ENV JAVA_7_HOME=/usr/lib/jvm/zulu-7-amd64
-ENV JAVA_9_HOME=/usr/lib/jvm/zulu-9-amd64
-ENV JAVA_10_HOME=/usr/lib/jvm/zulu-10-amd64
-ENV JAVA_11_HOME=/usr/lib/jvm/zulu-11-amd64
-ENV JAVA_12_HOME=/usr/lib/jvm/zulu-12-amd64
+    sudo apt-get install apt-transport-https socat; \
+    sudo apt-get install openjdk-8-jdk
 
-ENV JAVA_VERSION 1.8.0_sr5fp36
+RUN set -eux; \
```

Comment:
Why did you split this out into a separate `RUN` command?

Reply:
Readability sake, mainly